flux-kube/README.md

```diff
@@ -0,0 +1,25 @@
+### The Flux-Kube Unified Interface for Task submission
+
+This project provides a unified interface via Flux for 
+submission of HPC and K8s tasks which facilitates the use 
+of converged environments. This project is based on 
+Flux-framework and handles common K8s tasks along with 
+traditional HPC requests.
+
+The project has the following software dependencies:
+**Software**              | **version**             | **note**
+----------                | ----------              | --------
+Python                    | >= 3.6                  |
+OpenShift Python module   | >= 0.11.2               | install via `pip3 install openshift`
```

Comment:
Do you want to add a `requirements.txt` file?  That way you can embed the version information and simplify things to `pip install -r requirements`.  Otherwise, I'd set the pip command to something like `pip3 install openshift >= 0.11.2`

Reply:
Great suggestion, thanks!